clientapp/src/Api/PublishPsaRequest.js

```diff
@@ -9,7 +9,7 @@ export class PublishPsaRequest {
        this.params = params;
     }
 
-    getBody(cb) {
+    getBody(cb, errorCb) {
```

Comment:
This would certainly work too for now. currently, i'm handling async code with callbacks, but  I think at some point we want to move to promises, which is a lot cleaner and has the idea of success and error callbacks. Example:

`getBody.then(successCallback, errorCallback)`